foxtrot-core/pom.xml

```diff
@@ -113,19 +113,17 @@
                     <artifactId>guice</artifactId>
                     <groupId>com.google.inject</groupId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
                 <exclusion>
                     <groupId>com.google.inject.extensions</groupId>
                     <artifactId>guice-servlet</artifactId>
                 </exclusion>
             </exclusions>
         </dependency>
 
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>19.0</version>
-        </dependency>
```

Comment:
Removed? This has HBase dependency, enforce a specific version in dependencyManagement

Reply:
this was creating dependency conflict issues, so removed

Reply:
added guava 24.1.1-jre in dependency management